examples/internal/proto/examplepb/echo_service.swagger.json

```diff
@@ -5,6 +5,7 @@
     "description": "Echo Service API consists of a single service which returns\na message.",
     "version": "version not set"
   },
+  "tags": null,
```

Comment:
Could we omit this, rather than set it to null, in these instances?

Reply:
Yup. Fixed via `omitempty` in struct tags as suggested in other comments.